src/execute-js-func.js

```diff
@@ -108,15 +111,21 @@ function listenToIpc(channel) {
 function getSendMethod(windowOrWebView) {
   if (!windowOrWebView) return (...a) => ipc.send(...a);
 
-  return ('webContents' in windowOrWebView) ?
-    (...a) => {
+  if ('webContents' in windowOrWebView) {
+    return (...a) => {
       d(`webContents send: ${JSON.stringify(a)}`);
-      windowOrWebView.webContents.send(...a);
-    } :
-    (...a) => {
+      if (!windowOrWebView.webContents.isDestroyed()) {
+        windowOrWebView.webContents.send(...a);
+      } else {
+        d(`WebContents has been destroyed`);
```

Comment:
This should probably throw or else the caller gets a weird TimeoutError instead of the real error